packages/boot/tools/ibc/mocks.js

```diff
@@ -53,15 +29,21 @@ export const protoMsgMocks = {
 };
 
 /**
- * @param {string} versionString version JSON string
+ * Adds parameters to IBC version string if it's JSON
+ * @param {string} version version or JSON version string
  * @param {{ address: string; }} params
  * @returns {string}
  */
-const addParamsToVersion = (versionString, params) => {
-  return JSON.stringify({
-    ...JSON.parse(versionString),
-    ...params,
-  });
+export const addParamsToVersion = (version, params) => {
+  try {
+    const parsed = JSON.parse(version);
+    return JSON.stringify({
+      ...parsed,
+      ...params,
+    });
+  } catch {
+    return version;
```

Comment:
this silent failure looks like a bug risk. The caller should know what kind of `version` they have and whether `params` can be added. 

The fact that they passed `params` shows the expected it to be included. please revise

Reply:
I wouldn't consider this a silent failure. The idea is that if a `version` is a JSON string, add the `address` key. If it's not JSON (e.g. 'ics20-1'), don't change anything.

We can move this check inside `icaMocks.channelOpenAck` after `mockRemoteChannelID` if you'd prefer. I don't currently see a way to do the check without attempting a JSON.parse and seeing if it fails.

Reply:
>If it's not JSON (e.g. 'ics20-1'), don't change anything.

I see that. I'm saying that in that case, `addParamsToVersion` would not do what is says it does. This is just for mocks so a simple rename would suffice. E.g. `addParamsIfJsonVersion`

Reply:
Ok updated to `addParamsIfJsonVersion`